pkg/api/validation/validation_test.go

```diff
@@ -724,6 +724,10 @@ func TestValidatePodSpec(t *testing.T) {
 			RestartPolicy: api.RestartPolicyAlways,
```

Comment:
This applies to failure cases in other tests too, which is annoying and tedious to fix :(